src/Symfony/Component/Routing/Loader/schema/routing/routing-1.0.xsd

```diff
@@ -37,7 +37,8 @@
     <xsd:group ref="configs" minOccurs="0" maxOccurs="unbounded" />
 
     <xsd:attribute name="id" type="xsd:string" use="required" />
-    <xsd:attribute name="path" type="xsd:string" use="required" />
+    <xsd:attribute name="path" type="xsd:string" />
+    <xsd:attribute name="empty-path" type="xsd:boolean" />
```

Comment:
You could make a restriction on the boolean that it can only be true. Putting false there does not make sense.

```xml
<xsd:attribute name="empty-path" type="onlyTrue" />
  
  <xsd:simpleType name="onlyTrue">
    <xsd:restriction base="xsd:boolean">
      <xsd:enumeration value="true"/>
      <xsd:enumeration value="1"/>
    </xsd:restriction>
  </xsd:simpleType>
```

Reply:
with `type="only-true"` and this, I get `
type 'only-true': The facet 'enumeration' is not allowed on types derived from the type atomic type 'xs:boolean'.`

Reply:
OK, fixed